docs/documentation.md

```diff
@@ -122,12 +126,19 @@ The reason for using a container is that the NorESM model code is not an app but
+Input data is created using NCAR's `subset_data` [script to subset atmospheric forcing from large global data files](https://github.com/ESCOMP/CTSM/blob/master/tools/site_and_regional/subset_data). This is the simplest way, but has some limilations, specifically that restart runs are not supported. Read more [here](https://escomp.github.io/ctsm-docs/versions/release-clm5.0/html/users_guide/running-single-points/running-pts_mode-configurations.html), and see our Sites page for info on adding your own sites. Subsetting the data requires access to large, global files, and needs to be done on a supercomputer.
 
 The versioned input data are [in a shared folder on sigma2.no](https://ns2806k.webs.sigma2.no/EMERALD/EMERALD_platform/inputdata_noresm_landsites/). The .tar files are compressed and can be opened as a folder with e.g. 7-zip by right-clicking and choosing 'open archive', and used after extracting (unzipping). The data files are stored in [.nc (NetCDF)](https://www.unidata.ucar.edu/software/netcdf/) format, which can be viewed using [Panoply](https://www.giss.nasa.gov/tools/panoply/), or packages in Python or [R](https://cran.r-project.org/web/packages/ncdf4/index.html). The output data from simulations are stored in the same format but in the specific case folder.
 
+- **Legacy version**: An alternative, more manual way of creating input data is documented in the [NorESMhub/noresm-lsp-input](https://github.com/NorESMhub/noresm-lsp-input) repository. This method is similar to the subset_data method, i.e. there is a config file with the names of the datasets you want to subset, but instead of calling python scripts using xarray functionalities to subset the nearest neighbor of the grid cell values for domain/atm. forcing, this method relies on additional dependencies (perl etc.) and mapping/grid files, and requires access to large amounts of storage to create the necessary input data. This is because the global data files are very large and must be stored somewhere accessible while we subset from them. The scripts in the input repository are therefore run on e.g. Saga or another supercomputer. To use the legacy version, use the `legacy` branch of the noresm-land-sites-platform repository and make an .env file in the home folder where you add these two lines: 
```

Comment:
Just to avoid confusion, let's replace "home folder" with "project folder".

Reply:
done